submarine-sdk/pysubmarine/setup.py

```diff
@@ -15,11 +15,15 @@
 
 from setuptools import setup, find_packages
 
+with open('README.md') as f:
+    long_description = f.read()
 
 setup(
-    name='pysubmarine',
+    name='submarine-sdk',
```

Comment:
`PySubmarine` or `apache-submarine` ? I think it's not the `submarine-sdk`.

Reply:
I've updated it